### PR TITLE
proto: update import hooks, obey line length

### PR DESCRIPTION
**Pull request by exterkamp**
**Summary**
There are new, more strict proto linters now.  These hooks allow us to import easier.  And enforce the 80 char limit on our file here.

### CODE REVIEW

File: proto/lighthouse-result.proto [hunk 1]
```diff
@@ -1,10 +1,12 @@
 syntax = "proto3";
 
+// This comment required for Lightrider import compatibility # header
+
 import "google/protobuf/struct.proto";
 import "google/protobuf/timestamp.proto";
 import "google/protobuf/wrappers.proto";
 
-// This comment required for LightRider import compatibility
+// This comment required for LightRider import compatibility # suffix
```
Comment (brendankenny) [inline proto/lighthouse-result.proto]: i don't even

File: proto/lighthouse-result.proto [hunk 2]
```diff
@@ -142,7 +144,8 @@ message LighthouseResult {
     // nullable list of strings
     google.protobuf.Value only_categories = 3;
 
-    // How Lighthouse was run, e.g. from the Chrome extension or from the npm module
+    // How Lighthouse was run, e.g. from the Chrome extension or from the npm
+    // module
```
Comment (patrickhulce) [inline proto/lighthouse-result.proto]: ...

**Comment by connorjclark** [comment]
https://media.giphy.com/media/ZdObYXS0BQ1uYhAfWw/giphy.gif

File: proto/lighthouse-result.proto [hunk 3]
```diff
@@ -154,8 +157,8 @@ message LighthouseResult {
 
   // Message containing the performance timing data for the Lighthouse run
   message Timing {
-
-    // Corresponds to: https://www.w3.org/TR/performance-timeline-2/#dom-performanceentry
+    // Corresponds to:
```
Comment (connorjclark) [inline proto/lighthouse-result.proto]: heavy heavy heavy sigh

**Pull request merged**